### PR TITLE
Acronym guidelines

### PR DESCRIPTION
**Pull request by dblevins**
The only modifications of the document:

 - Addition of the phrase "Supplement to the Eclipse Foundation Guidelines for Eclipse Logos & T\
rademarks Policy for Oracle/JCP Acronyms" as found in the Trademark Guidelines
 - Reference turned into a hyperlink https://blogs.eclipse.org/post/mike-milinkovich/update-jakarta-ee-rights-java-trademarks
 - Acronym list was sorted

### CODE REVIEW

File: content/legal/acronym_guidelines/index.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+title: "Oracle/JCP Acronym Guidelines"
+date: 2019-06-26T16:10:38-04:00
+description: "Guidelines for the use of existing Oracle/JCP Acronyms.  Supplement to the Eclipse Foundation Guidelines for Eclipse Logos & Trademarks Policy for Oracle/JCP Acronyms"
```
Comment (bshannon) [inline content/legal/acronym_guidelines/index.md]: Where is this other document that this is a supplement to?

Comment (dblevins) [reply content/legal/acronym_guidelines/index.md]: Updated it to use the same link as this page: https://jakarta.ee/legal/trademark_guidelines/
That goes here: https://www.eclipse.org/legal/logo_guidelines.php

File: content/legal/acronym_guidelines/index.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+hide_sidebar: "false"
+---
+
+<p><em>Guidelines for the use of existing Oracle/JCP Acronyms.  Supplement to the Eclipse
+Foundation <a href="https://jakarta.ee/legal/trademark_guidelines/">Guidelines for Eclipse Logos &
+Trademarks Policy</a> for Oracle/JCP Acronyms</em></p>
```
Comment (bshannon) [inline content/legal/acronym_guidelines/index.md]: The "for Oracle/JCP Acronyms" at the end looks like it's part of the name of the supplement.
Really it just duplicates the first sentence.  Please remove it.

Comment (dblevins) [reply content/legal/acronym_guidelines/index.md]: Updated.

**Comment by autumnfound** [comment]
Hey David, it looks like you're missing the sign-off footer in some of your commits. Could you go back over them and just make sure all of your commits have the footer? Thanks!

**Comment by dblevins** [comment]
> Hey David, it looks like you're missing the sign-off footer in some of your commits. Could you go back over them and just make sure all of your commits have the footer? Thanks!

@autumnfound I attempted to do that and failed.  Do you have an instructions on how I can go back and fix them?

**Comment by chrisguindon** [comment]
@dblevins Here's some information about changing a commit message:
https://help.github.com/en/articles/changing-a-commit-message

Here's some information about Contributing via Git from the Eclipse Wiki:
https://wiki.eclipse.org/Development_Resources/Contributing_via_Git

**Comment by dblevins** [comment]
Humble suggestion.  Both of these links

 - https://help.github.com/en/articles/changing-a-commit-message
 - https://wiki.eclipse.org/Development_Resources/Contributing_via_Git

need to be in template that generates this message:

 - https://accounts.eclipse.org/legal/eca/validation/7501

If there's some way I can contribute that, I'll happily do it.

**Comment by waynebeaton** [comment]
> Humble suggestion. Both of these links

This is a good idea.

https://bugs.eclipse.org/bugs/show_bug.cgi?id=548775

**Comment by chrisguindon** [comment]
@waynebeaton there is already a bug for this https://bugs.eclipse.org/bugs/show_bug.cgi?id=546993

My team committed to improve error message and documentation for the ECA validation errors in Q3.

**Comment by chrisguindon** [comment]
@dblevins Thank you for updating your PR!

**Pull request merged**